### ISSUE TITLE
Blank screen when querying collection data

### ISSUE THREAD

**Issue by doanhieung** [zilliztech/attu#732]
**Describe the bug:**

When I attempted to view the data of a specific collection ("legal_kg_entities" in this case) within Attu, the interface becomes blank, so I can not browse and interact with the data stored in my Milvus collections through the Attu GUI.

![blank_screen](https://github.com/user-attachments/assets/b9b77b29-0fc1-4985-9ad1-f2aa01fd0393)

**Attu logs:**

```text
...
POST 200 /api/v1/collections/legal_kg_entities/query 90.175 ms @ Thu, 26 Dec 2024 11:00:34 GMT ::ffff:169.254.123.1 from undefined Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/131.0.0.0 Safari/537.36

ws client disconnected jdcink
```

**Steps to reproduce:**
1. Run Milvus in docker (Linux)
2. Insert some data
3. Run Attu in docker

**Attu version:** latest

```cmd
docker run -d -p 8000:3000 -e HOST_URL=http://127.0.0.1:8000 -e MILVUS_URL=10.130.0.2:19530 zilliz/attu:latest
```

**Milvus version:** 2.5.0

**Comment by shanghaikid** [comment]
what is you milvus version?

**Comment by shanghaikid** [comment]
can you show me your schema?

**Comment by doanhieung** [comment]
Yes

![image](https://github.com/user-attachments/assets/ec1f58a4-b156-4877-bbd5-bf2ad48d088c)

**Comment by shanghaikid** [comment]
> Yes
> 
> ![image](https://private-user-images.githubusercontent.com/137254384/398717938-ec1f58a4-b156-4877-bbd5-bf2ad48d088c.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzUyMTI4NTcsIm5iZiI6MTczNTIxMjU1NywicGF0aCI6Ii8xMzcyNTQzODQvMzk4NzE3OTM4LWVjMWY1OGE0LWIxNTYtNDg3Ny1iYmQ1LWJmMmFkNDhkMDg4Yy5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjI2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIyNlQxMTI5MTdaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0zZDlkYTBmNTVkOTAwOWJjOTViYWJiYWRiNjJmYjkxODAwZmE5Mzc4NjM1NzcxZjNlZmIyN2RjNTdjZTE3N2IxJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.e7PW98A05Lh-clW_0Lj4MJFuee_it4mmh_eSzYwd4i4)

Your screen is not attu 2.5.0. 

What's your attu version ? 
This is my test result: 
![image](https://github.com/user-attachments/assets/9369ec17-c2bd-4f46-a2ee-a4b0bcc948c1)
![image](https://github.com/user-attachments/assets/ec856bc6-0091-467a-9588-aa3afc46b6e5)

**Comment by doanhieung** [comment]
Yes, as stated in my first comment, I run Attu in docker with tag "latest" (and the GUI show version 2.4.10). 2.5.0 is Milvus version.

**Comment by shanghaikid** [comment]
> Yes, as stated in my first comment, I run Attu in docker with tag "latest" (and the GUI show version 2.4.10). 2.5.0 is Milvus version.

2.4.10 doesn't support Milvus. I just released attu v2.5.0 for Milvus 2.5. 

[v2.5.0](https://hub.docker.com/layers/zilliz/attu/v2.5.0/images/sha256-a156afcfbb6ed286ff0b36fb37d664f1956ea49bb983d6cee1b9175b20f4275e)

**Comment by doanhieung** [comment]
Ah, I see. Thank you.
